### PR TITLE
N31: Molecoole

### PR DESCRIPTION
**Pull request by Hilku**
Part of https://github.com/rust-gamedev/rust-gamedev.github.io/issues/949

### CODE REVIEW

File: content/news/031/index.md [hunk 1]
```diff
@@ -68,6 +68,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+Molecoole is a top-down shooter roguelike where you build your character from
+different atoms. Each atom has a unique ability to provide tons of variety
+between playthroughs. It's made using the [Bevy
+Engine](https://github.com/bevyengine/bevy).
+
+This month Molecoole launched on [Steam][molecoole-steam],
+making it the first game made with Bevy Engine there!
+It's available on Windows and Linux.
+You can check out it's gameplay on [youtube][youtube-molecoole],
```
Comment (17cupsofcoffee) [inline content/news/031/index.md:83]:
```suggestion
You can check out its gameplay on [youtube][youtube-molecoole],
```

File: content/news/031/index.md [hunk 2]
```diff
@@ -68,6 +68,25 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+or on the steam page. 
+
+
+[molecoole-steam]: https://store.steampowered.com/app/1792170/Molecoole/
```
Comment (17cupsofcoffee) [inline content/news/031/index.md:87]:
```suggestion
or on the Steam page. 

[molecoole-steam]: https://store.steampowered.com/app/1792170/Molecoole/
```

**Review by 17cupsofcoffee — APPROVED** [review]

File: content/news/031/index.md [hunk 3]
```diff
@@ -68,6 +68,24 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+Molecoole is a top-down shooter roguelike where you build your character from
+different atoms. Each atom has a unique ability to provide tons of variety
+between playthroughs. It's made using the [Bevy
+Engine](https://github.com/bevyengine/bevy).
+
+This month Molecoole launched on [Steam][molecoole-steam],
+making it the first game made with Bevy Engine there!
+It's available on Windows and Linux.
+You can check out its gameplay on [youtube][youtube-molecoole],
+or on the Steam page.
```
Comment (17cupsofcoffee) [inline content/news/031/index.md:84]:
```suggestion
or on the Steam page.
```

File: content/news/031/index.md [hunk 4]
```diff
@@ -68,6 +68,24 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Game Updates
 
+### [Molecoole][molecoole-steam]
+
+[![A molecoole and some enemies](molecoole.png)][molecoole-steam]
+
+Molecoole is a top-down shooter roguelike where you build your character from
+different atoms. Each atom has a unique ability to provide tons of variety
+between playthroughs. It's made using the [Bevy
+Engine](https://github.com/bevyengine/bevy).
```
Comment (AngelOnFira) [inline content/news/031/index.md:78]: Please move links to the bottom :)

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

**Review by AngelOnFira — APPROVED** [review]

**Pull request merged**